### DISCUSSION TITLE
rustdesk.exe FileDescription in Windows

### DISCUSSION THREAD

**Discussion by xlionjuan** [rustdesk/rustdesk#7381] [General]
The `FileDescription` will affect the name in task manager, if it isn't extend, there is no any name about RustDesk, it may should change to `RustDesk` or if you still want to keep that slogan, change it to `RustDesk - Open Source Remote Desktop Access Software`.
https://github.com/rustdesk/rustdesk/blob/56f27ec3569bb3278a8d707022276404ffbbb973/flutter/windows/runner/Runner.rc#L93

![截圖2024-03-14 17 50 13](https://github.com/rustdesk/rustdesk/assets/25243894/85e39160-83ce-4516-9e97-4f9f6fdc77c4)

Version: Today's 1.2.4 nightly

**Comment by rustdesk** [comment]
Thank you so much.
https://github.com/rustdesk/rustdesk/commit/175f185cb144005db7a7788decea739d6c09b0cf
